src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/OffenderService.kt

```diff
@@ -39,4 +40,47 @@ OffenderService(
       )
     }
   }
+
+  @Transactional
+  fun updateProbationRegion(crn: String, request: UpdateProbationTeamRequest) {
+    val inFlightLicenceStatuses = listOf(LicenceStatus.IN_PROGRESS, LicenceStatus.SUBMITTED, LicenceStatus.APPROVED, LicenceStatus.ACTIVE)
+    var offenderLicences = this.licenceRepository.findAllByCrnAndStatusCodeIn(crn, inFlightLicenceStatuses)
```

Comment:
Should we include VARIATION_IN_PROGRESS, VARIATION_SUBMITTED and VARIATION_APPROVED / REJECTED too?

Reply:
Oh! Yes we should, looks like we missed it on updating the COM too. Fixed now